### PR TITLE
Edit brevo contact

### PR DESCRIPTION
**Pull request by RainbowBunchie**
-   [x] Add changeset (if necessary)

Add an edit page for the contact 
Screenshot 2024-06-10 at 13 56 23
Screenshot 2024-06-10 at 13 57 06

### CODE REVIEW

File: demo/admin/src/common/brevoModuleConfig/brevoContactsPageAttributesConfig.tsx [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+            LASTNAME
+            FIRSTNAME
+            SALUTATION
+        }
+    }
+`;
+
+const salutationOptions: Array<{ label: React.ReactNode; value: GQLBrevoContactSalutation }> = [
+    {
+        label: <FormattedMessage id="targetGroup.filters.salutation.male." defaultMessage="Male" />,
```
Comment (raphaelblum) [inline demo/admin/src/common/brevoModuleConfig/brevoContactsPageAttributesConfig.tsx:26]:
```suggestion
        label: <FormattedMessage id="targetGroup.filters.salutation.male" defaultMessage="Male" />,
```

File: demo/admin/src/common/brevoModuleConfig/brevoContactsPageAttributesConfig.tsx [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+        value: "MALE",
+    },
+    {
+        label: <FormattedMessage id="targetGroup.filters.salutation.female." defaultMessage="Female" />,
```
Comment (raphaelblum) [inline demo/admin/src/common/brevoModuleConfig/brevoContactsPageAttributesConfig.tsx:30]:
```suggestion
        label: <FormattedMessage id="targetGroup.filters.salutation.female" defaultMessage="Female" />,
```

File: demo/admin/src/common/brevoModuleConfig/brevoContactsPageAttributesConfig.tsx (hunk 1 above)
Comment (RainbowBunchie) [reply demo/admin/src/common/brevoModuleConfig/brevoContactsPageAttributesConfig.tsx]: [e3e7156](https://github.com/vivid-planet/comet-brevo-module/pull/46/commits/e3e7156a9f4aeb96ad820258dd38f21b53c19133)

File: demo/admin/src/common/brevoModuleConfig/brevoContactsPageAttributesConfig.tsx (hunk 2 above)
Comment (RainbowBunchie) [reply demo/admin/src/common/brevoModuleConfig/brevoContactsPageAttributesConfig.tsx]: [e3e7156](https://github.com/vivid-planet/comet-brevo-module/pull/46/commits/e3e7156a9f4aeb96ad820258dd38f21b53c19133)

**Review by raphaelblum — DISMISSED** [review]

**Review by raphaelblum — APPROVED** [review]

**Pull request merged**